src/channel.cr

```diff
@@ -1,5 +1,21 @@
 require "fiber"
 
+# Channels enable concurrent communication between fibers.
+#
+#
+# ```
+# channel = Channel(Nil).new
+#
+# spawn do
+#   puts "Before send"
+#   channel.send(nil)
+#   puts "After send"
+# end
+#
+# puts "Before receive"
```

Comment:
The messages could be numbered to illustrate control flow.
```suggestion
# puts "1. Before receive"
```

Reply:
I fail to see the point of the whole example. The `puts` don't mean anything and are unerelated to Channel. In fact a buffered and unbuffered channel will behave differently. I would prefer the example to spawn one fiber that will send a few messages then let the main fiber receive those messages in the order they were sent, which is the most important thing.